src/dialects/maria/transaction.js

```diff
@@ -1,10 +1,11 @@
 
-var Transaction = require('../../transaction')
-var inherits    = require('inherits')
-var debug       = require('debug')('knex:tx')
-var helpers     = require('../../helpers')
-
+import inherits from 'inherits';
+import Debug from 'debug';
 import {assign} from 'lodash'
```

Comment:
in all new places in code I see, we have spaces around the words inside braces, i.e. according to that style this would be written as `import { assign } from 'lodash'`